frontend/README.md

```diff
@@ -4,7 +4,7 @@
 
 - [Yarn](https://classic.yarnpkg.com/lang/en/docs/install)
 - [GoLang 1.6](https://golang.org/doc/install)
-- [Node/NPM](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+- [Node/NPM v16](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
```

Comment:
👍 Thanks for adding the node version. I'm wondering if we should include `nvm` instructions so its easy to upgrade/downgrade node versions

Reply:
sure i'll update PR with the nvm instructions